src/test/java/nl/dtls/fairdatapoint/acceptance/actuator/List_Info_GET.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * The MIT License
+ * Copyright © 2017 DTL
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package nl.dtls.fairdatapoint.acceptance.actuator;
+
+import nl.dtls.fairdatapoint.WebIntegrationTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+
+import java.net.URI;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+public class List_Info_GET extends WebIntegrationTest {
+
+    private URI url() {
+        return URI.create("/actuator/info");
+    }
+
+    @Test
+    public void res200() {
+        // GIVEN:
+        RequestEntity<Void> request = RequestEntity
+                .get(url())
+                .build();
+        ParameterizedTypeReference<ActuatorInfoDTO> responseType = new ParameterizedTypeReference<>() {
+        };
+
+        // WHEN:
+        ResponseEntity<ActuatorInfoDTO> result = client.exchange(request, responseType);
+
+        // THEN:
+        assertThat(result.getStatusCode(), is(equalTo(HttpStatus.OK)));
```

Comment:
Do we need to check on content too, besides a `200`?

Reply:
It is possible, but that would duplicate the [AppInfoContributor](https://github.com/FAIRDataTeam/FAIRDataPoint/blob/fix/FDP-581/src/main/java/nl/dtls/fairdatapoint/service/actuator/AppInfoContributor.java). Alternatively, some extra annotations can be added to the class used for parsing the response (`@NotNull`, `@NotBlank`, or some regex pattern). Or both... What do you think?

Reply:
If it would catch a future change in spring, it might be convenient. But low priority.